src/lib/index.ts

```diff
@@ -57,3 +58,11 @@ export function makeSubtitle(html: string): string | null {
 		? stripHtmlTags(firstParagraph).slice(0, 100).trim() + '...'
 		: firstParagraph;
 }
+
+const authorTimestampFormat = "HH:mm EEE d MMMM yyyy"
```

Comment:
I'm not convinced that the exact time and day of the week is necessary

```suggestion
const authorTimestampFormat = "dd MMMM yyyy"
```

Reply:
Precise times can sometimes be helpful, and I don't see why we'd want to not provide info we have
Day of the week was an aesthetic thing - naked dates are a bit ugly I think